dwa_local_planner/cfg/DWAPlanner.cfg

```diff
@@ -25,6 +25,7 @@ gen.add("oscillation_reset_angle", double_t, 0, "The angle the robot must turn b
 
 gen.add("forward_point_distance", double_t, 0, "The distance from the center point of the robot to place an additional scoring point, in meters", 0.325)
 
+gen.add("scaling_discount_factor", double_t, 0, "Discount factor to apply to the scaling of the footprint for future points in the trajectory", 1.0, 0.0, 1.0)
```

Comment:
F: we have changed this then. Now 
`last scaled footprint = scaling_discount_factor * first scaled footprint`
so this is like a percentage